### DISCUSSION TITLE
数据副本放置在机架/机房的策略

### DISCUSSION THREAD

**Discussion by littleeleventhwolf** [apache/doris#8007] [A - General / Q&A]
Doris数据表，比如三副本，请问可以配置三副本放置在不同机架或机房的机器吗？

**Comment by morningman** [comment]
You can refer to this: http://doris.incubator.apache.org/zh-CN/administrator-guide/multi-tenant.html

**Reply by littleeleventhwolf** [reply]
还是有个问题：
如果我有4个(或更多)的机房，分别对应4个资源组：group_a, group_b, group_c, group_d，我怎么让三副本分别放置在任意不同的3个资源组里，peoperties中是像下面这样简单追加一个tag.location.group_d:1吗？

```
properties(
    "replication_allocation"
    =
    "tag.location.group_a:1, tag.location.group_b:1, tag.location.group_c:1, tag.location.group_d:1"
)
```
